gradle.properties.example

```diff
@@ -2,3 +2,6 @@
 #packerZip=https://releases.hashicorp.com/packer/1.3.4/packer_1.3.4_linux_amd64.zip
 #packerZip=https://releases.hashicorp.com/packer/1.3.4/packer_1.3.4_windows_amd64.zip
 #packerZip=https://releases.hashicorp.com/packer/1.3.4/packer_1.3.4_darwin_amd64.zip
+
+#Uncomment and set the path to the installBuilder installation directory
+#installBuilderHome=/opt/installbuilder-18.10.0
```

Comment:
I ran `./gradlew clean installerBuild --stacktrace`. Noticed that if the `installBuilderHome` path has a space in it, the command fails with: `Cannot run program ""/Applications/BitRock": error=2, No such file or directory`

 I've tried the following paths: 
```
installBuilderHome="/Applications/BitRock InstallBuilder Enterprise 19.5.0"
installBuilderHome="/Applications/BitRock\ InstallBuilder\ Enterprise\ 19.5.0
```